src/java.desktop/share/classes/javax/swing/ToolTipManager.java

```diff
@@ -265,13 +265,19 @@ void showTipWindow() {
                 toFind = new Point(screenLocation.x + preferredLocation.x,
                         screenLocation.y + preferredLocation.y);
             } else {
-                toFind = mouseEvent.getLocationOnScreen();
+                if (mouseEvent != null) {
+                    toFind = mouseEvent.getLocationOnScreen();
+                } else {
+                    toFind = screenLocation;
```

Comment:
Why `toFind` is set to `screenLocation` ?

Reply:
If mouse movement is not there, then it is imperative to use the original screenlocation to find GraphicsConfiuration but as told this codepath was not used so probably a no-op and should not be an issue..